README.md

```diff
@@ -40,7 +40,7 @@ Try out Solidus with one-click on Heroku:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/solidusio/solidus)
 
-## Getting started
+## Getting Started
```

Comment:
The contributing article says that the Solidus guides use sentence-style capitalization (`Getting started`) for article titles and subheadings, not title-style (`Getting Started`). See https://guides.solidus.io/contributing.html

If we change this, we should change all of the subheadings throughout the guides, and update the contributors guide as well.

Reply:
We can make the inverse change if you feel that is the right way to go -- I just tried to make the fewest changes possible based on the README. I'm all for being consistent when it comes to formatting 👍

Reply:
I see what you mean now. Carry on as you see fit. 💯

Reply:
I'd rather be consistent with the docs. Made that change. Thanks!